patterns/atoms/global/colors/_colors.scss

```diff
@@ -0,0 +1,28 @@
+
+.colors__color1 {
```

Comment:
Color classnames is not really ideal solution for applying colors, colors can be used in various properties: `color`, `background-color`, `border`, `box-shadow`, `linear-gradient()`, pseudo-classes `:hover`, `:visited`, `:link`, `:active` and so on. Color classnames approach requires to create class for every use case then. I would suggest to stick with semantic classnames (link in BEM methodology) which doesn't tell anything about how the element should look like, it just allows us to focus on CSS without touching markup. 

Speaking about drupal, it's easier to change color in CSS rather than change class on element.